src/bandersnatch/tests/test_utils.py

```diff
@@ -64,7 +65,11 @@ def test_rewrite(tmpdir, monkeypatch):
         f.write("csdf")
     assert open("sample").read() == "csdf"
     mode = os.stat("sample").st_mode
-    assert oct(mode) == "0o100644"
+    # chmod doesn't work on windows machines. Permissions are pinned at 666
+    if sys.platform == "win32":
+        assert True
+    else:
+        assert oct(mode) == "0o100644"
```

Comment:
All you need here is
```python
if not sys.platform == "win32":
    assert oct(mode) == "0o100644"
```

Reply:
Done. Using new utils.WINDOWS for the check.